package.json

```diff
@@ -70,8 +70,8 @@
         "posttest": "yarn lint",
         "prepack": "yarn build && oclif manifest && oclif readme",
         "prepare": "husky install",
-        "test": "export DISABLE_LOGS=true && mocha --forbid-only \"test/**/*.test.ts\"",
-        "test-logs": "mocha --forbid-only \"test/**/*.test.ts\"",
+        "test-no-logs": "export DISABLE_LOGS=true && mocha --forbid-only \"test/**/*.test.ts\"",
```

Comment:
@kartikp10 my thought here was that the logs can be noisy and by default we should suppress them. But now that I think about it more, it will only show warnings and errors by default, so that is probably useful. Anyways, your call.

Reply:
Running `export` fails the tests on windows - that's the only reason I flipped it. I can change it back and tweak the workflow steps

Reply:
Oh, yeah, my bad there. I think it's fine as you changed it then. It still puts all the failed tests cleanly at the end, after all the logs, which is what is most important